ANTORA.md

```diff
@@ -19,26 +19,37 @@ The Debezium Antora integration currently uses the [antora-default-ui](https://g
 ## Running Antora manually
 
 There are times where one may find it useful to regenerate just the Antora documentation while running the website in preview mode.
-In order to run Antora manually, simply open a bash session to the already running Jekyll docker container:
+
+You must first start the docker container for previewing the website, as described in this [README section](./README.md#22-using-the-container-image---generate-debezium-docs-from-local-repo).  If you want Antora to generate docs from your local debezium repo, it is **important** to include the volume mapping for the local debezium repo as described.
+
+Once the container is started, view the docker processes
+```
+docker ps -a
+```
+and copy the CONTAINER ID for the **debezium/website-builder** image.
```

Comment:
Would it be possible to assign a _name_ to the container so that there isn't the need for this extra step to acquire the random name that Docker has assigned?

Reply:
Updated the 'docker run' command in the README to specify a container name.  Then removed this section - no longer necessary to get the container id.